src/debugger/channel/DebugChannel.js

```diff
@@ -89,7 +89,12 @@ export class DebugChannel {
     this.writeOut(this._marshaller.marshallVariablesResponse(requestID, variables));
   }
 
-  sendPrepackFinish(): void {
-    this.writeOut(this._marshaller.marshallPrepackFinish());
+  sendStepInResponse(filePath: string, line: number, column: number) {
```

Comment:
Instead of sendStepInResponse(), I suggest you create a single debuggerStop message and with a field containing the stop reason(breakpoint, or step-into, step-over, step-out etc...). This aligns with VSP StoppedEvent.